docs/installation/macos.mdx

```diff
@@ -48,6 +48,12 @@ Specifically if your app is named 'MyApp', and are sharing your configuration be
 </dict>
 ```
 
+**Important** The next step is to enable `Outgoing Connections (Client)` on Xcode's `Runner.xcworkspace` as illustrated below:
```

Comment:
Is this emulator specific?

Reply:
I don't believe so actually. I was just following this [advice](https://github.com/FirebaseExtended/flutterfire/issues/6356#issuecomment-858560297) 😄. Can put under its own header if you wish?